client/src/legacy/GridField.js

```diff
@@ -175,8 +176,10 @@ $.entwine('ss', function($) {
     fixColumns: function (visible) {
       this.find('.sortable-header').append('<th class="main col-Actions" />');
       this.find('tbody tr').each(function () {
+        // Note we need to add 1 to the current column span, because we're going to add a new column
```

Comment:
Please move this comment down one as per the original suggestion in https://github.com/silverstripe/silverstripe-admin/pull/1869#discussion_r1887574735